src/scss/_custom.scss

```diff
@@ -182,14 +182,22 @@ input[type="password"]:not(:placeholder-shown) {
   position: relative;
 }
 
-.input-password .eye {
+.input-password .span {
```

Comment:
Didn't need to edit this line. Otherwise, it won't be selecting the eye from all the input passwords.